virtool/dispatcher/dispatcher.py

```diff
@@ -143,22 +143,22 @@ async def _dispatch(self, change: Change):
         :param change: the change to dispatch
 
         """
-        try:
-            fetcher = getattr(self._fetchers, change.interface)
-        except AttributeError:
-            logger.warning(f"Unknown dispatch interface: {change.interface}")
+        fetcher = getattr(self._fetchers, change.interface, None)
```

Comment:
Better to just `return` early if we encounter `AttributeError`:
```py
except AttributeError:
  logger.warning(f"Unknown dispatch interface: {change.interface}")
  return
```
This keeps the error and the resulting behaviour in the same place instead of using `None` variables and conditionals.